components/list/list-item.js

```diff
@@ -0,0 +1,183 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import ResizeObserver from 'resize-observer-polyfill';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+const ro = new ResizeObserver(entries => {
+	entries.forEach(entry => entry.target.resizedCallback(entry.contentRect));
+});
+
+class ListItem extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			_breakpoint: { type: Number }
+		};
+	}
+
+	static get styles() {
+		const layout = css`
+			.d2l-list-item-flex {
+				display: flex;
+			}
+			.d2l-list-item-content {
+				position: relative;
+				width: 100%;
+			}
+			.d2l-list-item-content-flex {
+				display: flex;
+				flex-grow: 1;
+				justify-content: stretch;
+				margin: 18px 0;
+				padding: var(--d2l-list-item-content-padding, 0);
+			}
+			::slotted([slot|="illustration"]){
+				align-self: flex-start;
+				display: flex;
+				flex-grow: 0;
+				flex-shrink: 0;
+				margin-right: 0.9rem;
+				max-height: 52px;
+				max-width: 90px;
+				overflow:hidden;
+			}
+			:host([dir="rtl"]) ::slotted([slot|="illustration"]) {
+				margin-left: 0.9rem;
+				margin-right: 0rem;
+			}
+			::slotted([slot="illustration-outer"]) {
+				margin-bottom: 18px;
+				margin-top: 18px;
+			}
+			::slotted([slot="actions"]) {
+				align-self: flex-start;
+				display: flex;
+				flex-grow: 0;
+			}
+			.d2l-list-item-main {
+				width: 100%;
+			}
+		`;
+		const mainContent = css`
+			::slotted(.d2l-list-item-text) {
```

Comment:
See comments in readme.

Reply:
Let's dig into this more, maybe outside of the PR.